docs/02_graphqlify.md

```diff
@@ -140,7 +141,7 @@ First we need to add an argument to our query and resolver.
 -   feed: User
 +   feed(username: String!): User
```

Comment:
Hm, these are actually literal diffs in the tutorial, not a change.

Reply:
🤔  weird